### PR TITLE
[VarDumper] Fix dumping Twig source in stack traces

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Branch? | 2.8 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #20320 |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/VarDumper/Tests/Fixtures/Twig.php [hunk 1]
```diff
@@ -5,9 +5,11 @@ class __TwigTemplate_VarDumperFixture_u75a09 extends Twig_Template
 {
     private $filename;
 
-    public function __construct(Twig_Environment $env, $filename = 'bar.twig')
+    public function __construct(Twig_Environment $env = null, $filename = null)
     {
-        parent::__construct($env);
+        if (null !== $env) {
```
Comment (hhamon) [inline src/Symfony/Component/VarDumper/Tests/Fixtures/Twig.php]: `if ($env)`

**Comment by javiereguiluz** [comment]
I've tested this PR and the new output is quite different from the one reported in #20320. However, I think that this new output makes it more difficult to understand what's going on (and there are some design issues, such as the different font families applied to the output):

deprecation_stack_trace

**Comment by nicolas-grekas** [comment]
@javiereguiluz please open a new issue or a PR (esp. for the font issue, which is not intentional)

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**